app/assets/javascripts/map.js

```diff
@@ -3,30 +3,53 @@ document.addEventListener("turbolinks:load", function () {
 
   if (mapEle) {
     const bigBenLatLng = [51.510357, -0.116773];
-    const map = L.map("map").setView(bigBenLatLng, 10);
 
-    const cercOptions = {
+    const osmBaseUrl = "https://ows.mundialis.de/services/service?";
+    const airtextBaseUrl = "https://airtext.info/geoserver/wms?";
```

Comment:
should be `airTextBaseUrl` I guess?

Reply:
thanks for spotting that offence against capitalisation